src/pages/get-started/release-notes.md

```diff
@@ -7,6 +7,20 @@ details: This page lists new features and known issues for each release of Adobe
 
 These release notes describe the latest version of Adobe I/O Events for Adobe Commerce.
 
+## Version 1.1.0
+
+### Release date
+
+April 20, 2023
+
+### Enchancements
+
+Added support for delivering events using message queues. Previously, all events were delivered by cron, which could delay delivery by up to a minute. Use the `bin/magento events:subscribe --priority` command to register the event as requiring expedited delivery. See [Configure Adobe Commerce](./configure-commerce.md/#check-cron-and-message-queue-configuration) for information about configuring the message queues.
```

Comment:
Also added a new feature, for field filtering for an array of nested objects.  

```
items[].sku
items[].qty
```
Previously it was possible only to add `items` field and that would return all content of `items` array without filtration.


https://github.com/AdobeDocs/commerce-events/pull/121/files#diff-4bbca1891cbffbe0985d7092411f8a79ff44bf43be9e65a769b0bd85ed93e8f5L215